### DISCUSSION TITLE
Undefined array key 0

### DISCUSSION THREAD

**Discussion by golchha21** [freekmurze/laravel-analytics#493] [General]
PFA the Screenshot
![image](https://github.com/spatie/laravel-analytics/assets/772400/ca567f51-4df9-4532-b602-6c2e972bb576)

I am getting the error "**Undefined array key 0**"

**Comment by deepvision7** [comment]
Could be related to GA4. Are you still using the old UA property which has already stopped processing data?

**Comment by golchha21** [comment]
@deepvision7 This has been solved. I was migrating form UA to GA4 and the way the data is generated had changed and was using the old key combination.
